.npmignore

```diff
@@ -0,0 +1,5 @@
+# Ignore everything
```

Comment:
thanks for introducing the .npmignore. Since the configuration isn self explanatory the comments don't seem necessary, could you remove them?